### ISSUE TITLE
add "*" indicator to the title of the YAML Editor

### ISSUE THREAD

**Issue by freegroup** [gardener/dashboard#218]
Maybe the commonly used **\*** is a good pattern to indicate that something has changed in the editor

## Unchanged editor

<img width="505" alt="bildschirmfoto 2018-10-15 um 11 23 00" src="https://user-images.githubusercontent.com/1155039/46942177-06dd9e00-d06d-11e8-9587-a197aab305bf.png">

## Editor with changes and indicator

<img width="537" alt="bildschirmfoto 2018-10-15 um 11 22 34" src="https://user-images.githubusercontent.com/1155039/46942220-1f4db880-d06d-11e8-83aa-13b045a574d2.png">

**Comment by grolu** [comment]
We discussed this also while we built the editor. The reason why we do not show a star next to the "YAML" tab is, that this would be semantically incorrect. The "YAML" tab is part of the navigation and not a file / cluster name. We show the dirty / edited state of the editor content by activating the save button on the left side of the toolbar.
However, we listen to our users. If we continue to get feedback regarding this, we will re-think the topic. If this is what most of the users would expect than we will add a star next to the "YAML" tab.
I will leave the ticket open for a while and wait if we get additional feedback.
